pkg/feature/servicemesh/resources.go

```diff
@@ -0,0 +1,20 @@
+package servicemesh
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/feature"
+)
+
+func ConfigMaps(feature *feature.Feature) error {
+	meshConfig := feature.Spec.ControlPlane
+	if err := feature.CreateConfigMap("service-mesh-refs",
+		map[string]string{
+			"CONTROL_PLANE_NAME": meshConfig.Name,
+			"MESH_NAMESPACE":     meshConfig.Namespace,
+		}); err != nil {
+		return errors.WithStack(err)
+	}
+
+	return nil
```

Comment:
@aslak I think we should populate `audiences` here too?

```suggestion
	meshConfig := feature.Spec.ControlPlane
	if err := feature.CreateConfigMap("service-mesh-refs",
		map[string]string{
			"CONTROL_PLANE_NAME": meshConfig.Name,
			"MESH_NAMESPACE":     meshConfig.Namespace,
		}); err != nil {
		return errors.WithStack(err)
	}

	authorinoConfig := feature.Spec.Auth.Authorino
	if err := feature.CreateConfigMap("auth-refs",
		map[string]string{
			"AUTH_AUDIENCE":   strings.Join(authorinoConfig.Audiences, ","),
		}); err != nil {
		return errors.WithStack(err)
	}
	
	return nil
```

Reply:
Oh right, this whole thing was from 605 of course..

Reply:
Shh! I just rebased everything today! :D

Reply:
Oh, well... I removed the ConfigMap in commit 540898 as I thought it wasn't needed.

Reply:
pushed https://github.com/opendatahub-io/opendatahub-operator/pull/784/commits/faede78d1d8669914fa556058ad804bcdc108c89